widget_driver/lib/src/widget_driver_provider.dart

```diff
@@ -13,4 +13,10 @@ abstract class WidgetDriverProvider<Driver extends WidgetDriver> {
 
   /// Creates and returns the `Driver` with the hard coded test values.
   Driver buildTestDriver();
+
+  /// This function only get's overriden when the driver has provided properties.
+  /// It calls a function in the driver, added by the `[SomeDriver]ProvidedPropertiesMixin`,
```

Comment:
```suggestion
  /// It calls a method in the driver, added by the `[SomeDriver]ProvidedPropertiesMixin`,
```

I think in general dart refers to functions as methods